src/Employer.Web/Configuration/IoC.cs

```diff
@@ -16,24 +18,45 @@ public static void AddIoC(this IServiceCollection services, IConfiguration confi
             //Configuration
             services.Configure<ExternalLinksConfiguration>(configuration.GetSection("ExternalLinks"));
             services.Configure<AuthenticationConfiguration>(configuration.GetSection("Authentication"));
+            services.Configure<AccountApiConfiguration>(configuration.GetSection("AccountApiConfiguration"));
+            
+            RegisterAccountApiClientDeps(services);
```

Comment:
I'm liking extensions for stuff like this so services.AddAccountApiClientDeps();

Reply:
I probably wouldn't for the moment as I think we are changing stuff or well I will be. But have no issue with sticking it in extension methods later on. I can understand where you have for the Recruit client thingy.